### DISCUSSION TITLE
台製無人機更勝土製無人機

### DISCUSSION THREAD

**Discussion by PoooYen** [1101-GS4719-C/Python-Community-GS4719-C#480] [General]
除了飛彈部署數量驚人，本刊掌握，中科院還有許多祕密武器未公布，如研發多年的火紅雀無人機，外型類似迷你版的螺旋槳飛機，不但具有人工智慧，可透過人臉辨識執行斬首行動，更能自主編隊、掃描、追蹤和發動蜂群式攻擊，形成鋪天蓋地的火力網，宛如電影《全面攻佔3：天使救援》，只要遭無人機鎖定即難逃追擊，勢將成為國軍新世代的作戰利器。

https://www.msn.com/zh-tw/news/national/%E5%B0%96%E7%AB%AF%E6%AD%A6%E5%99%A8%E8%AD%B7%E5%8F%B02-%E7%A7%91%E5%B9%BB%E9%9B%BB%E5%BD%B1%E5%A5%AA%E5%91%BD%E8%A1%80%E6%BB%B4%E5%AD%90%E6%88%90%E7%9C%9F-ai%E7%84%A1%E4%BA%BA%E6%A9%9F%E7%BE%A4%E6%94%BB%E6%88%90%E6%96%AC%E9%A6%96%E5%88%A9%E5%99%A8/ar-AAV6vqc?ocid=msedgntp
![image](https://user-images.githubusercontent.com/99785984/158542883-9df0e329-4dca-483d-a9f2-6ea1df7c1f96.png)

由這項科技的發布我們可以看到人臉的圖像辨識已經進步到可以用來作軍事用途，台灣的政治及地緣關係非常適合這種高精準打擊的武器，成本小效益高，希望日後也可以在銀行領錢時或政府機關洽公時可以使用人臉領錢或轉帳或簽名，減少實體資源的使用，而且台灣的軍事研發實力不會輸給別國，看看紅火雀的數據就可以知道，希望大家可以對台灣有信心，一起為這塊美麗的土地付出。

**Comment by sean1325418** [comment]
台灣人口不多，這種無人機可以有效解決這問題，並且也能拿來執行更多元的戰術，這種東西越多越好。

**Comment by xkllkx** [comment]
在台灣這個人口密集的地方，無人機最適合了，將來如果開放更多民間應用，生活會更便利。

**Comment by hapiwangy** [comment]
平時真的不太會接觸到這類的資訊，有了無人機的幫忙，對於人口不多的地方來說，真的是一大福音

**Comment by ghost** [comment]
科技發展固然是一大福音，但執政者如何利用又是一項課題就是了

**Comment by Yuhsin000** [comment]
覺得這是很好拿來保衛家園的東西，台灣小型的軍用設備其實不少的感覺

**Comment by cordydeng** [comment]
擁有足夠的軍事設備對國家安全來說是一個保障 , 但還是希望不會有用到的一天

**Comment by awkward-willy** [comment]
科技的發展對戰爭的整體影響還是很顯著的
這次的烏俄戰爭正可以體現這點
不過，戰爭還是應當被避免
畢竟到頭來可憐的都是同為人類的彼此

**Comment by treeann** [comment]
真的相當厲害ㄝ 無人跡

**Comment by etin2468** [comment]
我是覺得台灣發展這種軍用的技術好像助益不大拉，我是希望可以多把這些錢投入科研

**Comment by minniewei** [comment]
雖然說是有必要對軍事方面事務進行投資，但還是希望不要發生戰爭....

**Comment by sys511613** [comment]
人臉辨識的精準度也是一大挑戰
現在手機的faceid就常常打不開了

**Comment by leon10456** [comment]
感謝樓主分享

**Comment by jcpan821** [comment]
但拿來作戰還沒飛到對岸就沒電了吧

**Comment by Lynn12378** [comment]
比起軍事，感覺運用在民生可以帶來更多好處
